lib/core/test/shared/Cardano/Wallet/DummyTarget/Primitive/Types.hs

```diff
@@ -80,16 +79,12 @@ instance PersistTx DummyTarget where
     mkTx _ inps = Tx (fst <$> inps)
 
 instance KeyToAddress DummyTarget SeqKey where
-    keyToAddress = Address . unXPub . getRawKey
+    keyToAddress =
+        Address . BS.take 8 . convertToBase Base16 . unXPub . getRawKey
 
 instance KeyToAddress DummyTarget RndKey where
-    keyToAddress k = Address
-        $ CBOR.toStrictByteString
-        $ CBOR.encodeAddress (getRawKey k)
-            [ CBOR.encodeDerivationPathAttr pwd acctIx addrIx ]
-      where
-        (acctIx, addrIx) = derivationPath k
-        pwd = payloadPassphrase k
+    keyToAddress =
+        Address . BS.take 8 . convertToBase Base16 . unXPub . getRawKey
```

Comment:
Converting to base16 and using only the first 8 bytes for more readability in test failures! 
At the DB-level, addresses are really just a bunch of bytes, so no need to maintain any sort of CBOR structure there.